src/components/Dashboard/TalkDashboard.vue

```diff
@@ -0,0 +1,226 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<script lang="ts" setup>
+import { ref, onMounted, computed, onBeforeUnmount } from 'vue'
+import { useRouter } from 'vue-router/composables'
+
+import IconAt from 'vue-material-design-icons/At.vue'
+import IconCalendarBlank from 'vue-material-design-icons/CalendarBlank.vue'
+import IconMicrophone from 'vue-material-design-icons/Microphone.vue'
+import IconVideo from 'vue-material-design-icons/Video.vue'
+
+import { showError } from '@nextcloud/dialogs'
+import { emit } from '@nextcloud/event-bus'
+import { t } from '@nextcloud/l10n'
+
+import NcButton from '@nextcloud/vue/components/NcButton'
+import NcEmptyContent from '@nextcloud/vue/components/NcEmptyContent'
+
+import EventCard from './EventCard.vue'
+import ConversationsListVirtual from '../LeftSidebar/ConversationsList/ConversationsListVirtual.vue'
+import LoadingPlaceholder from '../UIShared/LoadingPlaceholder.vue'
+
+import { useStore } from '../../composables/useStore.js'
+import { CONVERSATION } from '../../constants.ts'
+import { useTalkDashboardStore } from '../../stores/talkdashboard.ts'
+import type { Conversation, DashboardEventRoom } from '../../types/index.ts'
+import { filterConversation } from '../../utils/conversation.ts'
+
+const FIVE_MINUTES = 5 * 60 * 1000 // 5 minutes
+const store = useStore()
+const router = useRouter()
+const talkDashboardStore = useTalkDashboardStore()
+
+const eventRooms = ref<DashboardEventRoom[]>([])
+const loading = ref(false)
+const fetchEventRoomsInterval = ref<ReturnType<typeof setInterval> | null>(null)
+
+onMounted(async () => {
+	fetchEventRooms()
+	fetchEventRoomsInterval.value = setInterval(fetchEventRooms, FIVE_MINUTES)
+})
+
+onBeforeUnmount(() => {
+	if (fetchEventRoomsInterval.value) {
+		clearInterval(fetchEventRoomsInterval.value)
+	}
+})
+
+const conversationsList = computed(() => store.getters.conversationsList)
+const conversationsInitialised = computed(() => store.getters.conversationsInitialised)
+const filteredConversations = computed(() => conversationsList.value?.filter((conversation : Conversation) => {
+	return filterConversation(conversation, ['mentions'])
+}))
+
+/**
+ * Fetches the event rooms
+ */
+async function fetchEventRooms() {
+	try {
+		loading.value = true
+		eventRooms.value = await talkDashboardStore.fetchDashboardEventRooms() ?? []
+		loading.value = false
+	} catch (error) {
+		console.error('Error fetching event rooms:', error)
+		showError(t('spreed', 'Error fetching event rooms'))
+	}
+}
+
+/**
+ * Creates a new group conversation and navigates to the conversation page.
+ */
+async function startMeeting() {
+	try {
+		const conversation = await store.dispatch('createGroupConversation', {
+			roomName: t('spreed', 'Meeting'), // NOTE: it uses user language statically
+			roomType: CONVERSATION.TYPE.PUBLIC,
```

Comment:
```suggestion
			roomType: CONVERSATION.TYPE.PUBLIC,
			objectType: 'instant_meeting',
```

Align with https://github.com/nextcloud/spreed/pull/15051 and use `'conversation-creation-all'` capability (just in case)

Reply:
>  use 'conversation-creation-all' capability

it's implemented inside the store action